.travis.yml

```diff
@@ -89,39 +93,46 @@ install:
   ;;
   esac
 - cargo install --list
+
 script: |
   case "$FEATURE" in
 
   "fmt" )
-      cd exonum && cargo update && cd .. &&
-      cd sandbox && cargo update && cd .. &&
-      cd exonum && cargo audit && cd .. &&
+      cargo update &&
       cd exonum && cargo fmt -- --write-mode=diff && cd .. &&
-      cd sandbox && cargo fmt -- --write-mode=diff && cd ..
+      cd sandbox && cargo fmt -- --write-mode=diff && cd .. &&
+      cd testkit && cargo fmt -- --write-mode=diff && cd ..
   ;;
   "clippy-core" )
-      cd exonum && cargo clippy --verbose --features long_benchmarks  -- -D warnings
+      cd exonum && cargo clippy --verbose --features long_benchmarks -- -D warnings
   ;;
   "clippy-sandbox" )
       cd sandbox && cargo clippy --verbose -- -D warnings
   ;;
+  "clippy-testkit" )
+      cd testkit && cargo clippy --verbose -- -D warnings
+  ;;
   "test-core" )
-      cargo test --manifest-path exonum/Cargo.toml --verbose --tests --lib --features "float_serialize"
+      cargo test -p exonum --verbose --tests --lib --features "float_serialize"
   ;;
   "test-sandbox" )
-      cargo test --manifest-path sandbox/Cargo.toml  --verbose
+      cargo test -p sandbox --verbose
+  ;;
+  "test-testkit" )
+      cargo test -p exonum-testkit --verbose
```

Comment:
What about `cargo test --all` and `cargo build --all`?

Reply:
Unfortunately, `cargo ... --all` isn't working because of nightly features in `3rdparty/flamer`.